### DISCUSSION TITLE
Slinger Player Audio Choppy on one computer all of a sudden

### DISCUSSION THREAD

**Discussion by RayMan412** [GerryDazoo/SlingerPlayer#37] [General]
Optiplex 5050
Slinger Player worked fine for long long time.
All of a sudden audio choppy in Slinger Player - HOWEVER in VLC it's fine.
Reinstalled windows, reinstalled everthing....same result.
Any ideas?

**Comment by GerryDazoo** [comment]
Might be a network issue is this on the LAN or WAN. VLC maintains a much larger internal buffer to smooth out glitches. SlingerPlayer uses minimal buffering to improved remote control operation.
For a quick test, once the video starts click in the video window and this will pause the playback click again after a few seconds and see if the jitter goes away,
(Oops, I may have broken this feature on the latest update. you may have to backup to the previous release to do this test.)
